pages/common/mpg321.md

```diff
@@ -0,0 +1,33 @@
+# mpg321
+
+> High Performance MPEG 1.0/2.0/2.5 Audio Player for Layer 1, 2, and 3.
+> Mpg321 was written (sometime in 1999) to be a drop-in replacement for the (previously) non-free mpg123 player.
+> More information: <http://mpg321.sourceforge.net/>.
+
+- Play an audio source exactly N=3 times:
+
+`mpg321 -l {{3}} {{path/to/file_a|URL}} {{path/to/file_b|URL}} {{...}}`
+
+- Play an audio source forever (0 means until interrupted):
+
+`mpg321 -l 0 {{path/to/file_a|URL}} {{path/to/file_b|URL}} {{...}}`
+
+- Play a whole folder (recursively):
+
+`mpg321 -B {{path/to/directory}}`
+
+- Enable *Basic Keys* ( `*` or `/` - Increase or decrease volume, `n` - Skip song, `m` - Mute/unmute.) while playing:
+
+`mpg321 -K {{path/to/file_a|URL}} {{path/to/file_b|URL}} {{...}}`
+
+- Play files randomly until interrupted:
+
+`mpg321 -Z {{path/to/file_a|URL}} {{path/to/file_b|URL}} {{...}}`
+
+- Shuffle list of files before playing the list once:
+
+`mpg321 -z {{path/to/file_a|URL}} {{path/to/file_b|URL}} {{...}}`
+
+- Play all the files in the current folder(s) and subfolders, randomly (until interrupted), with *Basic Keys* enabled:
```

Comment:
We usually leave the combination of multiple options to the users themselves. It is expected that a command will work by combining the individual flags. Wondering if it still makes sense to have this?

Reply:
Sincerely, yes.... actually this is MY main reason for creating this tldr page - I didn't remember the flags, and I always had to look into its man docs for the flags.... and this is what I expect from tldr: to give me - useful examples.
Besides, what would be the gain in removing this examples?! that we'd have one line less in this file? I strongly suggest to leave this example as it is!

Reply:
From what I could understand, the flags are already covered in the previous examples. We just need to combine them to get the functionality desired.

My reasoning was to remain consistent with the rest of the pages in the repo. Because this is what we usually follow and encourage our users to build up combos themselves. But it's fine if you want to keep it. Thanks.

Reply:
Thank you, too!